ui/main/src/app/views/activityarea/activityarea.view.ts

```
@@ -27,6 +27,8 @@ export class ActivityAreaView {
         string,
         ActivityAreaEntityCluster
     >();
+    private setClusterLineCheckBoxValue: Function = () => {};
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)